### ISSUE TITLE
Google Lighthouse audit results

### ISSUE THREAD

**Issue by missmatsuko** [tmirmota/reactvancouver#11]
This can be separated into different issues, but here are some areas to improve based on Lighthouse audit results:

## Performance: 29%
- Can save 30+ seconds by fixing #8.

## Accessibility: 89%
- Chat `iframe` missing `title` (maybe can't fix, I think it's added from an external script)
- Several links don't have a discernible name:
  - "Get involved" in the header is actually a `<button>` in a `<a>`. The text in the button can stay, but the button shouldn't be there. "Discover speakers" has the same issue. Also, should probably change the date styles so they don't look like buttons.
  - AxiomZen linked logo image missing alt text (should be `AxiomZen`)
  - Slack and Meetup linked icons missing screenreader text (by the way, why not link the email address that's right above it, too?)
- Credit text at the bottom of the site has insufficient contrast
- `<html>` missing `lang` attribute

## Best practices: 71%
- There's a warning for using `document.write()` but probably can't do much about that because it's coming from `messenger.js`
- Add `rel="noopener"` or `rel="noreferrer"` to any external links (the AxiomZen link)
- Lo-Dash@4.17.10 identified as having 2 vulnerabilities
- Error logged to console: `Error: Smallchat initialized without required config properties.`
